release-notes.txt

```diff
@@ -1,3 +1,23 @@
+OpenMDAO Version 1.7.1 Alpha Release Notes
+July 18, 2016
+
+Features:
+* Add an `xtol` option to Newton. This ceases iteration when the Newton step shrinks below a threshold.
+* Better user_controllable solver convergence printing (iprint).
```

Comment:
The `print_all_convergence` function in Probem now has an argument level that lets you choose between: 0 -- only display failures 1 -- display iteration counts 2 -- display residuals each iteration (this is the default)